src/integration/pages/onDemandTVPage/pashto.expired/canonical.test.js

```diff
@@ -4,9 +4,19 @@
  */
 
 import runExpiredEpisodeTests from '../expiredEpisodeTests';
-import runCanonicalTests from '../canonicalTests';
+import runCommonEpisodeTests from '../commonEpisodeTests';
+import {
+  runCoreCanonicalTests,
+  runCanonicalAnalyticsTests,
+  runCommonCrossPlatformTests,
+  runRecentEpisodesTests,
+} from '../../../common';
 
 describe('Canonical Pashto On Demand TV Page', () => {
   runExpiredEpisodeTests();
-  runCanonicalTests();
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+  runCommonCrossPlatformTests(service);
+  runCommonEpisodeTests();
+  runRecentEpisodesTests();
```

Comment:
I think we can revert this, because the canonical tests in https://github.com/bbc/simorgh/pull/11798/files#diff-680c81936f3c974920b24c54b4048b1e2bc800f4908383f62beb40de508d9a03 are already running the cross platform stuff

```suggestion
  runCanonicalTests();
```